src/Engine/Graphics/Outdoor.cpp

```diff
@@ -2321,6 +2321,8 @@ void TeleportToStartingPoint(MapStartPoint point) {
             for (size_t i = 0; i < pLevelDecorations.size(); ++i) {
                 if (pLevelDecorations[i].uDecorationDescID == pDecorationList->GetDecorIdByName(pName)) {
                     pParty->pos = pLevelDecorations[i].vPosition;
+                    // Spawn point in Harmondale from Barrow Downs is up in the sky, vanilla worked it around by
+                    // always placing the party on the ground.
```

Comment:
This is good info but it goes beyond what PR says.

Reply:
Also, I suggest we mark important notes, especially such deviations, with `NOTE` - other comments explaining code are fine without.

Reply:
We're taking this to discord for now, I'm happy to add a `NOTE` there later.